### ISSUE TITLE
JS resources at top are blocking page

### ISSUE THREAD

**Issue by macmladen** [Dejanzr/Deve-Assigment#8]
Eps as they refer to CDN, they should be all on bottom, except which one?

**Comment by Dejanzr** [comment]
Mladen, thank you for all pointers. Except jQuery ?

**Comment by macmladen** [comment]
No, Google analytics is the only one that should stay in the head, everything else goes to bottom in order they are required so jQuery before those who depend on it.

_____________

Mladen Djuric
Front end / Drupal Developer
Develomon

Phone +381 63 509 289
Skype macmladen
develomon.com

**Comment by Dejanzr** [comment]
Will remember that, thank you.

**Comment by Dejanzr** [comment]
May I  just ask ,  beside i didn't done in sass but instead in css. Did i meet any demands of the task you gave?
